packages/mux-player/src/template.ts

```diff
@@ -195,6 +225,24 @@ export const VodChromeLarge = (props: MuxTemplateProps) => html`
   </media-control-bar>
 `;
 
+// prettier-ignore
+export const LiveChromeExtraSmall = (props: MuxTemplateProps) => html`
```

Comment:
nit (blocking): If these are identical (for now), let's share the template fn as:
`export const LiveChromeExtraSmall = VodChromeExtraSmall;`